validator/project_validator.go

```diff
@@ -982,6 +982,12 @@ func validateCommands(section string, project *model.Project,
 				Message: fmt.Sprintf("cannot specify both command '%s' and function '%s'", cmd.Command, cmd.Function),
 			})
 		}
+		if cmd.Command == "shell.exec" && cmd.Params["script"] == nil {
+			errs = append(errs, ValidationError{
+				Level:   Error,
```

Comment:
An error will break the build of a project entirely that may have been working before.  I think since having no script is not a critical error we should downgrade this to a warning, just to notify the user but not break their build.